explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+Yes, the API is designed to help facilitate communication of PII from secure wallet applications to verifier web servers. 
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+Today, online identity verification (e.g., for KYC) is usually done by submitting photos of government identity documents (for example, uploading photos containing the PII via the web’s `<input type=file>` mechanism. Due to the privacy and security limits of this approach, credential issuers and verifiers are working to move to the use of wallet applications which can selectively disclose cryptographically-attested properties. Today, those approaches rely on generic web→app communication paths like custom schemes and server-to-server communication. 
+
+The use of wallets aims to improve on the status quo of uploading images in a number of ways:
+ * Enable selective disclosure (e.g., for more privacy-friendly age≥18 verification)
+ * Enable end-to-end PII encryption between the wallet application and the relevant verification server
+ * Enable request authentication, where a wallet is required to cryptographically verify that the requester has permission to access the credential
+
+Further, this specification aims to improve over the existing communication channels used by websites to talk to wallet apps by:
+ * Enabling browsers and operating systems to provide meaningful credential selection and permission screens to users, prior to wallet applications becoming aware of the presentation request
+ * Securely communicating the origin of the requesting site to the wallet application, so that it can implement its own MITM protection with support of WebPKI
+ * Enabling browsers to inspect requests and provide additional UI affordances to users
+
+> 05.  Do the features in your specification introduce state
+>      that persists across browsing sessions?
+
+No. In particular, the current scope is just about credential presentation (read-only). We have had requests to expand scope to consider credential issuance APIs but no work has begun on that.
+
+> 06.  Do the features in your specification expose information about the
+>      underlying platform to origins?
+
+Not directly. Wallets may (intentionally or inadvertently) expose some information through this API (such as some indication of which wallet app the user is using). But the primary motivation for these wallets over traditional federated authentication systems is that the wallet can act in the user’s interest to protect their privacy even from the credential issuer. Issuers generally choose which wallets to support their credentials in and can impose privacy requirements on those wallets. Users will often have their choice of multiple wallets and are expected to use reputation for privacy as part of their decision making.
```

Comment:
This paints a very optimistic picture / best case scenario of wallets doing the right thing. 

Although from the outset this says states that wallets "may (intentionally or inadvertently) expose some information through this API", we should probably outline a worst case scenario. 

Like, this could speak about the response data being standardized to be structured in a particular way (even if the browser can't directly peek at the encrypted response data).

Reply:
Yeah I can emphasize the risk more. I also added a statement referring to discussion in #117.